README.md

```diff
@@ -0,0 +1,71 @@
+# GitHub App for Issue Resolution
+
+> This GitHub App is powered by [Aider](https://aider.chat/) and [Claude 3.5 Sonnet](https://www.anthropic.com/news/claude-3-5-sonnet).
```

Comment:
Let's also add a section above the 'setup instructions' to show 'how it works'. This should include:
- how it listens for the GitHub events - when a new issue is created, when a new PR review comment is created
- how it clones the repo and uses aider to run an LLM prompt that makes the changes
- how it pushes its changes to a new branch
- how it uses the GitHub API to create new PRs, create comments etc.